lib/workos/audit_logs/audit_logs.ex

```diff
@@ -0,0 +1,166 @@
+defmodule WorkOS.AuditLogs do
+  import WorkOS.API
+
+  @moduledoc """
+  The Audit Logs module provides convenience methods for working with the
+  WorkOS Audit Logs platform. You'll need a valid API key.
+
+  See https://workos.com/docs/audit-logs
+  """
+
+  @doc """
+  Create an Audit Log Event.
+
+  ### Parameters
+  - params (map)
+    - organization (string) The unique ID of the Organization
+     that the event is associated with.
+    - event (map) The Audit Log event
+
+  ### Examples
+
+      iex> WorkOS.AuditLogs.create_event(%{
+      ...>  organization: "org_123",
+      ...>  event: %{
+      ...>    action: "user.signed_in",
+      ...>    occurred_at: "2022-09-08T19:46:03.435Z",
+      ...>    version: 1,
+      ...>    actor: %{
+      ...>      id: "user_TF4C5938",
+      ...>      type: "user",
+      ...>      name: "Jon Smith",
+      ...>      metadata: %{
+      ...>        role: "admin"
+      ...>      }
+      ...>    },
+      ...>    targets: [
+      ...>      %{
+      ...>        id: "user_98432YHF",
+      ...>        type: "user",
+      ...>        name: "Jon Smith"
+      ...>      },
+      ...>      %{
+      ...>        id: "team_J8YASKA2",
+      ...>        type: "team",
+      ...>        metadata: %{
+      ...>          owner: "user_01GBTCQ2"
+      ...>        }
+      ...>      }
+      ...>    ],
+      ...>    context: %{
+      ...>      location: "New York, NY",
+      ...>      user_agent: "Chrome/104.0.0"
+      ...>    },
+      ...>    metadata: %{
+      ...>      extra: "data"
+      ...>    }
+      ...>  }
+      ...> })
+
+      {:ok, nil}
+
+  """
+
+  def create_event(params, opts \\ [])
+
+  def create_event(params, opts)
+      when is_map_key(params, :organization) and is_map_key(params, :event) do
+    post(
+      "/audit_logs/events",
+      %{
+        organization_id: params.organization,
+        event: params.event
+      },
+      opts
+    )
+  end
+
+  def create_event(_params, _opts),
+    do: raise(ArgumentError, message: "Missing required parameters: organization, event")
+
+
+  @doc """
+  Create an Export of Audit Log Events.
+
+  ### Parameters
+  - params (map)
+    - organization (string) The unique ID of the Organization
+     that the event is associated with.
+    - range_start (string) ISO-8601 value for start of the export range.
+    - range_end (string) ISO-8601 value for end of the export range.
+    - actions (list) List of actions to filter against.
```

Comment:
Should this be noted as a "list of strings" specifically? I'm not sure how much this matters, because of dynamic typing.

Reply:
yea I wasn't sure if there was a specific way to define the type in the list or if I could just put `(list of strings)`